lib/backend/registrybackend/security/security.go

```diff
@@ -173,6 +175,12 @@ func (c credentialStore) Basic(*url.URL) (string, string) {
 	if basic == nil {
 		return "", ""
 	}
+	if c.config.PasswordFile != "" {
+		passwordBytes, err := ioutil.ReadFile(c.config.PasswordFile)
+		if err == nil {
+			return basic.Username, (string)(passwordBytes)
```

Comment:
In Go, it's preferred to typecast by calling _T(v)_, where _T_ is the new type and _v_ is the value:
`return basic.Username, string(passwordBytes)`